WORKSPACE

```diff
@@ -34,14 +34,16 @@ switched_rules_by_language(
 
 maven_install(
     artifacts = IO_GRPC_GRPC_JAVA_ARTIFACTS + PROTOBUF_MAVEN_ARTIFACTS,
-    generate_compat_repositories = True,
     override_targets = IO_GRPC_GRPC_JAVA_OVERRIDE_TARGETS,
     repositories = [
         "https://repo.maven.apache.org/maven2/",
     ],
+    maven_install_json = "@//:maven_install.json",
```

Comment:
Let's revert this and the following changes. The lack of pinning hasn't really been a problem and having pinning will create problems. We can remove the compat_repositories though.